### DISCUSSION TITLE
Guest/players see when players got banned/jailed

### DISCUSSION THREAD

**Discussion by Fornuften** [Zrips/CMI#956] [Enhancement]
I have a problem that players see when other players get banned / jailed. I have looked at perms / config. but can not figure out who permission node players may have or not. Also, I do not find anything in config either. Is there a solution to this? I do not want ordinary players to see when people are banned / jailed 

Cmi version 8.0.19.0 
spigot 
This server is running CraftBukkit version git-Spigot-417cfb3-4487c1f (MC: 1.13)

**Comment by CloudeLecaw** [comment]
you know you can take the -s Option for These commands?
like /ban Fornuften -s
and no worry..this ban sees noone not even your admins ;)

There is no permisison for hiding (that allow to Show) the Texts just for Special Groups.
if you Need this feel free to create a Suggestion with this. Until this you have the -s :)

**Comment by glorying16** [comment]
I too would like to be able to hide/allow by using permissions. I dont want the players to see those things but I do want my staff to see it. Using -s would only be enticing to me if I also didnt want my staff to see it. Im thinking if you set the ban/mute/jail etc to be picked up by your social spy, then maybe edit the message to nothing that would alert staff, but no one else? Just a thought for a temporary fix. I really hope this gets added though.

**Comment by CloudeLecaw** [comment]
So the answer is no for the Moment, this is not possible currently. It is no Bug to fix because it never was made like this. You may can hole like all other suggestion that this may come into cmi one time or not :)  I recommand to change the title with [suggestion] infront or create a New one and close thread :)

**Comment by glorying16** [comment]
Just wondering if this still a possibility?
